lib/cmd/fh3/projects/create.js

```diff
@@ -0,0 +1,61 @@
+/* globals i18n */
+var common = require("../../../common");
+var ini = require('../../../utils/ini');
+var templates = require('../../common/templates.js');
+var util = require('util');
+
+
+module.exports = {
+  'desc' : i18n._('Create project'),
+  'examples' :
+  [{
+    cmd : 'fhc projects create --title=<title>',
+    desc : i18n._('Create a  with the <title>')
+  },{
+    cmd : 'fhc projects create --title=<title> --template=<template>',
+    desc : i18n._('Create a project with the <title> and <template>')
+  },{
+    cmd : 'fhc projects create --title=<title> --template=<template> --env=<environment>',
+    desc : i18n._('Create a project with the <title> and <template>, after deploy it on <environment>')
+  }],
+  'demand' : ['title'],
+  'alias' : {
+    'title' : 't',
+    'template' : 't',
```

Comment:
@wei-lee Tks for the review and to finding it. It doesn't will work and will fix it.